kyo-prelude/shared/src/main/scala/kyo2/Maybe.scala

```diff
@@ -6,8 +6,8 @@ import Maybe.internal.*
 opaque type Maybe[+A] >: (Empty | Defined[A]) = Empty | Defined[A]
 
 object Maybe:
-    given [A, B](using CanEqual[A, B]): CanEqual[Maybe[A], Maybe[B]] = CanEqual.derived
-    given [A]: Conversion[Maybe[A], IterableOnce[A]]                 = _.iterator
+    inline given [A, B](using inline ce: CanEqual[A, B]): CanEqual[Maybe[A], Maybe[B]] = CanEqual.derived
```

Comment:
Inlining to avoid Scala's lazy initialization of `given` values. It's also a "free" inlining since it doesn't increase bytecode size (a method call getting replaced by another)